airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/activities/RefreshSchemaActivityTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling.activities;
+
+import static org.mockito.Mockito.mock;
+
+import io.airbyte.config.ActorCatalogFetchEvent;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.workers.temporal.sync.RefreshSchemaActivityImpl;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class RefreshSchemaActivityTest {
+
+  static private ConfigRepository mConfigRepository;
+  static private RefreshSchemaActivityImpl refreshSchemaActivity;
+
+  static private final UUID SOURCE_ID = UUID.randomUUID();
+
+  @BeforeEach
+  void setUp() {
+    mConfigRepository = mock(ConfigRepository.class);
```

Comment:
Nit we shouldn't mix static import and no import on the same file. It should be `Mockito.mock` or line 39 should be `when` instead of `Mockito.when`.